src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  color: $secondary-text;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  padding-top: 32px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid $card-border;
+  transition-duration: $trans-duration;
```

Add transtion property as well, use short hand
```suggestion
  transition: transform $trans-duration;
```